precompiles/ics20/events.go

```diff
@@ -16,7 +16,7 @@ const (
 	// EventTypeIBCTransfer defines the event type for the ICS20 Transfer transaction.
 	EventTypeIBCTransfer = "IBCTransfer"
 	// EventTypeIBCTransferAuthorization defines the event type for the ICS20 TransferAuthorization transaction.
-	EventTypeIBCTransferAuthorization = "IBCTransferAuthorization"
+	EventTypeIBCTransferAuthorization = "IBCTransferAuthorization" //nolint:gosec
```

Comment:
would probably be better to specifically escape the gosec warning
```suggestion
	EventTypeIBCTransferAuthorization = "IBCTransferAuthorization" //#nosec GXXX
```